src/views/RichWorkspace.vue

```diff
@@ -175,6 +175,7 @@ export default {
 		text-align: left;
 		max-height: 0;
 		transition: max-height 0.5s cubic-bezier(0, 1, 0, 1);
+		z-index: 110;
```

Comment:
The cypress test failure seems expected due to this z-index being larger than the one of the file creation menu which would cause some overlapping like this:

![image](https://user-images.githubusercontent.com/3404133/133295272-793b67ed-53ff-4367-be67-0d2c2cea2ed5.png)

In addition it might be good to add a `position: relative;` here as well to make sure the z-index always applies.